tutorials/tutorial10/tutorial.py

```diff
@@ -0,0 +1,112 @@
+"""Tutorial of the Averaging Neural Operator."""
```

Comment:
Maybe for the tutorial we could think to solve a nice NO problem, such as the KdV equation 1D, what do you think? Also would you like to add the tutorial in the same PR, in that case we need the ipynb, rst, ... let's think about it together!

Reply:
Do you already have a dataset about this PDE?

Reply:
Yes! sharing with you in private